package.json

```diff
@@ -176,6 +177,7 @@
     "extract-text-webpack-plugin": "3.0.2",
     "favicons-webpack-plugin": "0.0.7",
     "file-loader": "1.1.5",
+    "font-awesome": "4.7.0",
```

Comment:
Can bootstrap and font-awesome use the `^` char? Probably the last Bootstrap 3.x version, but the latest font-awesome is still v4.7.0